Templates/Biostats_template.Rmd

```diff
@@ -127,8 +135,9 @@ p
 
 ```
 
-Number the figures in the text in a logical order: e.g. Figure 4.1
-And refer to each figure in the text as: Figure 4.1
+The RMarkdown file will take care of the numbering of the figures (bookdown html).
+
+The figures can be referd to in the text using @ref(fig:display-figure)
```

Comment:
needs to be a backslack before @ (I think) problem is markdown will probably eat it

Reply:
no it works like this. when I add a \ it only shows the figure number...

Reply:
maybe put backticks round it so the backslash shows up rather than just getting the figure number `\@ref(fig:my_fig)`

Reply:
and "referred"